### DISCUSSION TITLE
BigQuery Integration

### DISCUSSION THREAD

**Discussion by ZackSam** [Budibase/budibase#3393] [Ideas]
Hi Budibase, 

I found a thread from February 2021 mentioning BigQuery in the pipeline. Have there been any updates? 

A data warehouse solution would be the cherry on top for Budibase as I would like to transform multiple DBs into supertables for our education use case. 

Thanks again for all the incredible work!

**Comment by mjashanks** [comment]
Hey @ZackSam.

Agree, this would be a great integration. However, it's not going to be a priority for the core team over the next few months. We would welcome any contributions for this. 

It probably wouldn't be that difficult for someone with BigQuery experience, here is the MongoDb integration for example: https://github.com/Budibase/budibase/blob/develop/packages/server/src/integrations/mongodb.ts
